assets/blocks/course-outline/course-outline-block.js

```diff
@@ -0,0 +1,32 @@
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+import { InnerBlocks } from '@wordpress/block-editor';
+
+registerBlockType( 'sensei-lms/course-outline', {
+	title: __( 'Outline', 'sensei-lms' ),
+	description: __( 'Manage your Sensei LMS course outline.', 'sensei-lms' ),
+	icon: 'list-view',
+	category: 'sensei-lms',
+	keywords: [ __( 'Outline', 'sensei-lms' ), __( 'Course', 'sensei-lms' ) ],
+	supports: {
+		html: false,
+		customClassName: false,
```

Comment:
Any reason to disable `customClassName`?

Reply:
Oops! Good catch! It was supposed to be only for the lessons and modules. https://github.com/Automattic/sensei/pull/3560/commits/f805a4410de92da374f8ca3d6c864b601adc6a83

But thinking again, I'm not sure if we should remove it from the lessons and modules, or try to handle that. Maybe we could leave them as `customClassName: false,` and revisit it while implementing the listings. Do you agree?